DataProcessing/datax-core/pom.xml

```diff
@@ -50,7 +50,7 @@ SOFTWARE
   </scm>
 
   <groupId>com.microsoft.datax</groupId>
-  <artifactId>datax-core_3.1_2.12</artifactId>
+  <artifactId>datax-core_3.1_2.13</artifactId>
   <version>1.2.17-SNAPSHOT</version>
```

Comment:
The version bump will be from 1.2.17-SNAPSHOT to 1.2.18-SNAPSHOT. The 2.12 is the scala version

Reply:
fixed. thanks for the pointer.